### PR TITLE
Pmm 12766  - Adds Rel notes for  newMongoDb dashboards

### PR DESCRIPTION
**Pull request by catalinaadam**
PMM-0

Link to the Feature Build: SUBMODULES-0

If this PR adds or removes or alters one or more API endpoints, please review and add or update the relevant [API documents](https://github.com/percona/pmm/tree/main/docs/api) as well:

- [ ] API Docs updated

If this PR is related to some other PRs in this or other repositories, please provide links to those PRs:

- Links to related pull requests (optional).

### CODE REVIEW

**Review by YashSartanpara1 — APPROVED** [review]

File: docs/release-notes/2.43.0.md [hunk 1]
```diff
@@ -10,9 +10,45 @@ It enables you to observe the health of your database systems, explore new patte
...
+
+
+This update includes significant improvements to MongoDB monitoring focused on two key areas: the general availability (GA) of redesigned dashboards and a streamlined navigation structure for easier access to important dashboards.
+
+#### Redesigned dashboards
+
+The following MongoDB dashboards, previously in Experimental status, have been redesigned and are now generally available:
+
+- **MongoDB Instance Summary**
+- **Replica Set Summary**
```
Comment (rnovikovP) [inline docs/release-notes/2.43.0.md:25]: we now use the one dashboard for Replica Set Summary and Instance Summary

File: docs/release-notes/2.43.0.md [hunk 2]
```diff
@@ -10,9 +10,45 @@ It enables you to observe the health of your database systems, explore new patte
...
+- **Sharded Cluster Summary** (formerly Cluster Summary)
+
```
Comment (rnovikovP) [inline docs/release-notes/2.43.0.md:27]: there are several more dashboards moved from Experimental to  GA

**Comment by rnovikovP** [comment]
and the Image file from the OS structure. we need a MongoDB one

**Review by rnovikovP — APPROVED** [review]
but image - looks strange 🤔

**Pull request merged**